dev-cpp/ut2-glaze/ut2-glaze-0.0.4.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# it's a header-only lib, thus not cmake-multilib. However, examples and tests...
+inherit cmake
+
+DESCRIPTION="A stripped down fork of boost-ext ut2 "
+HOMEPAGE="https://github.com/openalgz/ut"
+
+SRC_URI="
+	https://github.com/openalgz/ut/archive/refs/tags/v${PV}.tar.gz -> ut2-openalgz-${PV}.tar.gz"
+
+S="${WORKDIR}/ut-${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~arm64-macos ~ppc-macos ~x64-macos ~x64-solaris"
+IUSE="test doc"
+
+# REQUIRED_USE=""
+
+RESTRICT="!test? ( test )"
+
+# PROPERTIES=""
+
+# DEPEND=""
+
+# RDEPEND="${DEPEND}"
+
+BDEPEND="
+dev-build/cmake
+"
+
+PATCHES=(
```

Comment:
Same comment here for patches, please explain why and where they're from if you are not the author

Reply:
I can understand the motivation behind those patches which most of them are quite straightforward, and it would be great if we can push them back to upstream, so we will eventually carry less..

for 0001_change_project_name_to_ut2-glaze.patch, I'm not sure if it's really necessary.. is there  actually any conflict? or just rename to make it more glaze specific? since it's a fork?